pep-0517.txt

```diff
@@ -285,11 +231,66 @@ specifies UTF-8 based file names. This is not yet the default for the tarfile
 module shipped with Python 3.6, so backends using the tarfile module need to
 explicitly pass ``format=tarfile.PAX_FORMAT``.
 
-This must return the basename (not the full path) of the ``.tar.gz`` file it
+The hook must return the basename (not the full path) of the ``.tar.gz`` file it
 creates, as a unicode string.
```

Comment:
Same note as for `build_wheel`: I think we should either move this up to be the second paragraph in the hook description, or else insert "in ``sdist_directory``" after "creates"